docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -157,7 +157,7 @@ def build(self):
     def package(self):
         copy(self, pattern="LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         autotools = Autotools(self)
-        # TODO: replace by autotools.install() once https://github.com/conan-io/conan/issues/12153 fixed
+        # TODO: replace by autotools.install() once Conan 1.54 is available in CCI
```

Comment:
```suggestion
        # TODO: replace by autotools.install() if required_conan_version greater or equal than 1.54
```

Reply:
The templates should generally just be the latest. As per other convo - we dont except contributors to know the different version. If they do that's amazing ❤️ 

But I'd rather not confuse any who doesn't know what version is available in CCI versus what they can install locally :)

For that I will not be apply the suggestion - hopefully that makes sense

Reply:
I don't understand your reasoning.

Reply:
I am afraid  Contributors won't understand you suggestion.. most are aware of the difference between what Conan release, what CCI is build and what the templates are written for

If that assertion is wrong its not our expectation or a requirement for them too and the docs need to be written as such